tests/nnfw_api/lib/CircleGen.cc

```diff
@@ -596,6 +596,13 @@ uint32_t CircleGen::addOperatorRmsNorm(const OperatorParams &params, float epsil
                                 circle::BuiltinOptions_RmsNormOptions, options);
 }
 
+uint32_t CircleGen::addOperatorRoPE(const OperatorParams &params)
```

Comment:
@ys44kim Could you please add `mode` param?

Reply:
Currently, only one mode (GPT-NEOX) is supported, so I thought the default value was used. i didn't care about mode.
I argee it's good to add mode like your opinion.
i will add `mode`.